source/includes/_files.md

```diff
@@ -0,0 +1,199 @@
+# Files
+
+On the OpenDataSoft platform, files can be used as data source to create new datasets.
+
+## The file object
+
+> Example file object
+
+```json
+{
+    "file_id": "cheese_data.csv",
```

Comment:
Don't we want to use file_uid ? (only in the API no migration needed).
To be consistent with processor_uid, dataset_uid, change_uid.
@jmorel  ?